app/assets/stylesheets/pages/about.scss

```diff
@@ -0,0 +1,118 @@
+.about-page  {
+  margin-top: 60px;
+  max-width: 1024px;
+  margin-bottom: 48px;
+
+  /* headers and text styling */
+
+  .about-intro {
+    .intro-title {
+      font-size: 18px;
+      height: 24px;
+      font-weight: bold;
+      font-style: normal;
+      font-stretch: normal;
+      line-height: 1.33;
+      letter-spacing: 0.3px;
+      color: $text-title;
+      margin-left: 6.5%;
```

Comment:
seems like a lot of these should be baseline properties that are inherited or applied here with a global level class. Like if across the application a title should look like this, then we can just set `h2` or a `title` class or something and reuse it everywhere.

Reply:
@uuunk I think It'll be nice if we can talk about this some more during the meeting. I know I have a lot of similar repetitions in my current pr and previous ones. It'll be nice if we had a good understanding on how to take care of this. I put up a [question](https://trello.com/c/w76pRctU/51-need-to-figure-out-a-good-way-to-handle-font-styling) about it some time ago but I don't know if we talked about it last time.